src/lib.rs

```diff
@@ -92,8 +92,16 @@ use solana_jsonrpc_core as jsonrpc_core;
 use solana_jsonrpc_http_server as jsonrpc_http_server;
 #[macro_use]
 extern crate solana_jsonrpc_macros as jsonrpc_macros;
-use solana_jsonrpc_pubsub as jsonrpc_pubsub;
-use solana_jsonrpc_ws_server as jsonrpc_ws_server;
+extern crate solana_jsonrpc_pubsub as jsonrpc_pubsub;
+extern crate solana_jsonrpc_ws_server as jsonrpc_ws_server;
+extern crate solana_metrics;
+extern crate solana_native_loader;
```

Comment:
All these `extern crate`s can probably go. This looks like a bad rebase on the Rust 2018 updates.